x-pack/plugins/actions/server/builtin_action_types/lib/request_oauth_jwt_token.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import qs from 'query-string';
+import axios from 'axios';
+import stringify from 'json-stable-stringify';
+import { request } from '../lib/axios_utils';
+import { Logger } from '../../../../../../src/core/server';
+import { ActionsConfigurationUtilities } from '../../actions_config';
+
+export const OAUTH_JWT_BEARER_GRANT_TYPE = 'urn:ietf:params:oauth:grant-type:jwt-bearer';
```

Comment:
Is this from some standard? Maybe a link to that standard or a comment about what this means would be helpful?